Build/Scripts/UpdateDependencies.bat

```diff
@@ -1,7 +1,7 @@
 ::*******************************************************************************************************
 ::  UpdateDependencies.bat - Gbtc
 ::
-::  Copyright � 2013, Grid Protection Alliance.  All Rights Reserved.
+::  Copyright � 2013, Grid Protection Alliance.  All Rights Reserved.
```

Comment:
This doesn't seems to be a valid UTF-8 symbol. The original code point was U+00A9, which seems to be the correct character for representing the copyright symbol. Should we revert it?

Reply:
Actually, looks like this file is ANSI-encoded, not Unicode. So the code point was just 0xA9. We could convert this character to UTF-8 instead.